pkg/kube/longhorn-generate-support-bundle.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+LOG_DIR=/persist/newlog/kube
+
+echo "Apply longhorn support bundle yaml at $(date)"
+
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: longhorn.io/v1beta2
+kind: SupportBundle
+metadata:
+  name: support-bundle-collect-info
+  namespace: longhorn-system
+spec:
+  description: collect-info
+  issueURL: ""
+  nodeID: ""
+EOF
+
+ready=0
+while [ $ready -ne 1 ]; do
+    state=$(kubectl -n longhorn-system get supportbundle.longhorn.io/support-bundle-collect-info -o json | jq -r .status.state)
+    if [ "$state" = "ReadyForDownload" ]; then
+        ready=1
+        break
```

Comment:
If you are using break you don't need the ready variable.... you can just use `while [ 1 ]; do`

Reply:
oh yes, that is unnecessary